### PR TITLE
Fix 802

### LINKED ISSUE #802: ERROR building surface (MS, SAS or AV) for structure 6X5S
@fredludlow 
This is something strange happening only on structure 6X5S (tested MS, SAS or AV type and filterSelection ":A". This configuration should reproduce the error immediately, but it randomly happens also without filterSelection). I tested Firefox and Chrome, and they give the same results.   

```
Uncaught TypeError: Cannot read property 'getPosition' of undefined
    at MolecularSurfaceRepresentation.createData (molecularsurface-representation.js:180)
    at structure-representation.js:165
    at Array.forEach (<anonymous>)
    at MolecularSurfaceRepresentation.create (structure-representation.js:161)
    at _make (representation.js:368)
    at after (molecularsurface-representation.js:159)
    at molecularsurface-representation.js:168
    at onSurfaceFinish (molecularsurface-representation.js:130)
    at Worker.<anonymous> (molecular-surface.js:99)
    at Worker.worker.onmessage (worker.js:24)
```

On AV surface only an additional error is shown in the console 

```
worker.js:47 Worker.onerror molsurf 
ErrorEvent {isTrusted: true, message: "Uncaught RangeError: Invalid typed array length: Infinity", filename: "blob:null/954c0b75-9607-4543-8378-42b357e0e646", lineno: 2453, colno: 25, …}
bubbles: false
cancelBubble: false
cancelable: true
colno: 25
composed: false
currentTarget: Window {window: Window, self: Window, document: document, name: "", location: Location, …}
defaultPrevented: false
error: null
eventPhase: 0
filename: "blob:null/954c0b75-9607-4543-8378-42b357e0e646"
isTrusted: true
lineno: 2453
message: "Uncaught RangeError: Invalid typed array length: Infinity"
path: [Window]
returnValue: true
srcElement: Window {window: Window, self: Window, document: document, name: "", location: Location, …}
target: Window {window: Window, self: Window, document: document, name: "", location: Location, …}
timeStamp: 27043.87499999939
type: "error"
__proto__: ErrorEvent
```

### PR DESCRIPTION
**Pull request by fredludlow**
As the title suggests, should fix #802 by avoiding trying to build a surface from an empty `StructureView` instance.

### CODE REVIEW

**Review by giagitom — APPROVED** [review]

**Comment by giagitom** [comment]
Thanks @fredludlow, this seems to work fine for me.

**Comment by fredludlow** [comment]
Great - @arose, would you be able to review/approve as appropriate?

**Comment by giagitom** [comment]
Hi, Any news about this?

**Comment by fredludlow** [comment]
It either needs a review from a "member" I think, or for the owner to unilaterally make the change. I'll request the relevant people, but it would be great to have some more people authorized to approve things (e.g. yourself @giagitom) as many PRs get stalled at the moment. 

@arose - I wonder if it would also be worth making a couple of us owners as we're a bit stuck in limbo at the moment without a critical mass of reviewers/maintainers to keep things moving.

**Review by garyo — APPROVED** [review]
Looks good to me!

**Comment by fredludlow** [comment]
Thankyou @garyo!

**Pull request merged**
